src/components/BookDisplay.js

```diff
@@ -1,20 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 
 const BookDisplay = (props) => {
-  const { Title, Author } = props;
+  const { Title, Author, Id } = props;
+
+  const dispatch = useDispatch();
+  const handleRemove = () => {
+    dispatch(removeBook(Id));
+  };
```

Comment:
The code for removing a book from the state looks correct ✅ , but there's a small mistake in the `removeBook` reducer. 😢  Instead of using `book.id`, which doesn't exist in the `book` object, ❌  you should use `book.item_id` to compare with `action.payload`. 🤔 

Here's the corrected `removeBook` reducer:
```
removeBook(state, action) {
  return state.filter((book) => book.item_id !== action.payload);
},
```